DashKit/DashPumpManagerState.swift

```diff
@@ -187,12 +190,23 @@ public struct DashPumpManagerState: RawRepresentable, Equatable {
             self.finishedDoses = []
         }
         
-        if let rawActiveAlerts = rawValue["activeAlerts"] as? PodAlerts.RawValue {
-            self.activeAlerts = PodAlerts(rawValue: rawActiveAlerts)
+        if let rawActiveAlerts = rawValue["activeAlerts"] as? [PumpManagerAlert.RawValue] {
+            self.activeAlerts = []
+            for rawAlert in rawActiveAlerts {
+                if let alert = PumpManagerAlert(rawValue: rawAlert) {
+                    self.activeAlerts.insert(alert)
+                }
+            }
         } else {
             self.activeAlerts = []
         }
         
+        if let acknowledgedTimeOffsetAlert = rawValue["acknowledgedTimeOffsetAlert"] as? Bool {
```

Comment:
[nit] Consider changing to `self.acknowledgedTimeOffsetAlert = rawValue["acknowledgedTimeOffsetAlert"] as? Bool ?? false`